tests/tasks/test_api_complete_task.py

```diff
@@ -41,6 +43,54 @@ def test_complete_task_api(self):
         self.assertIsNotNone(complete_task.get('completed_at'))
         self.assertTrue(complete_task.get('is_done'))
 
+    def test_achieve_task_token_missing(self):
```

Comment:
These tests not in table are actually meant for accepting mentorship relation without token and with expired one.Can you update the pr to reflect the same?

Reply:
So the task listing is wrong? It says "Valid user tries to accept valid task with authentication token missing"

Reply:
I'll do it but i just want to be sure that I understand well

Reply:
Ohh my bad :sweat_smile: Will update the task description.Can you add these tests for now?

Reply:
Sure, I will!

Reply:
Should I remove these 2 (testing task completion without token and with expired token)?

Reply:
Sorry for the late reply.Approving it for now but can you remove these tests?

Reply:
Okay, I removed them :)